runtime/ts/storage/pouchdb/pouch-db-storage-provider.ts

```diff
@@ -23,7 +23,7 @@ export abstract class PouchDbStorageProvider extends StorageProviderBase {
   /** The Pouch revision of the data we have stored locally */
   protected _rev: string | undefined;
 
-  constructor(type: Type, storageEngine: PouchDbStorage, name: string, id: string, key: string) {
+  protected constructor(type: Type, storageEngine: PouchDbStorage, name: string, id: string, key: string) {
```

Comment:
Does `protected` for constructors imply that the class can only be constructed by itself or a subclass?